media-proxy/include/mesh/conn.h

```diff
@@ -66,7 +66,7 @@ enum class Result {
     error_bad_argument,
     error_out_of_memory,
     error_general_failure,
-
+    error_shutdown,
```

Comment:
```suggestion
    error_context_cancelled,
```

Reply:
Fixed in 7b57d140d1ff8be6328bad2c1e5c18c53ad97bae